### PR TITLE
[website] Add a Catalog page for the GluonNLP trained models

### PR DESCRIPTION
**Pull request by StrayBird-ATSH**
## Description ##
[website] Add a Catalog page for the GluonNLP trained models

### Essentials ###
- [x] PR's title starts with a category (e.g. [BUGFIX], [MODEL], [TUTORIAL], [FEATURE], [DOC], etc)
- [ ] Changes are complete (i.e. I finished coding on this PR)
- [ ] All changes have test coverage
- [ ] Code is well-documented

### Changes ###
- [ ] Feature1, tests, (and when applicable, API doc)
- [ ] Feature2, tests, (and when applicable, API doc)

## Comments ##
- If this change is a backward incompatible change, why must this change be made.
- Interesting edge cases to note here

cc @dmlc/gluon-nlp-team

### CODE REVIEW

**Comment by mli** [comment]
Job PR-1184/1 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-1184/1/index.html

File: scripts/catalog/index.rst [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Catalog for GluonNLP
```
Comment (eric-haibin-lin) [inline scripts/catalog/index.rst]: Can we put everything together in 1 page instead of separate ones?

Comment (StrayBird-ATSH) [reply scripts/catalog/index.rst]: OK, I will make corresponding changes

**Comment by mli** [comment]
Job PR-1184/3 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-1184/3/index.html

File: scripts/catalog.rst [hunk 2]
```diff
@@ -0,0 +1,28 @@
+Catalog for GluonNLP
```
Comment (eric-haibin-lin) [inline scripts/catalog.rst]: rename it to Model Catalog

File: scripts/catalog.rst [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
++-------------------------------------------------------------+----------------------------------+-----------------+------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| 3-layer AWD-LSTM language model (#hidden units = 1150)      | awd_lstm_lm_1150_wikitext-2      | 65.62           |python word_language_model.py --gpu 0 --tied --ntasgd --lr_update_interval 30 --lr_update_factor 0.1 --save awd_lstm_lm_1150_wikitext-2                                                                                                                                                                                                                                             |
```
Comment (eric-haibin-lin) [inline scripts/catalog.rst]: remove 1st column, and add reference to the paper

File: scripts/catalog.rst [hunk 4]
```diff
@@ -0,0 +1,28 @@
+Catalog for GluonNLP
+====================
+
+
+
+
+Language Model
+--------------
+https://gluon-nlp.mxnet.io/model_zoo/language_model/index.html
```
Comment (eric-haibin-lin) [inline scripts/catalog.rst]: is it possible to use relative link?

**Comment by mli** [comment]
Job PR-1184/4 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-1184/4/index.html

**Comment by mli** [comment]
Job PR-1184/5 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-1184/5/index.html

**Comment by StrayBird-ATSH** [comment]
Currently, I am writing [this page](https://github.com/StrayBird-ATSH/gluon-nlp/blob/patch-2/scripts/catalog.rst) the same style as [gluon cv](https://gluon-cv.mxnet.io/model_zoo/segmentation.html).
I think that we are using policies for warnings: they let warnings go but we treat warnings as [errors](https://ci.mxnet.io/blue/organizations/jenkins/GluonNLP-py3-master-gpu-doc/detail/PR-1184/8/pipeline/).

So for the last two columns, do you want to change CI to make them display the same **command**, or change the **rst** code to let them display different contents?

Thanks!

**Comment by mli** [comment]
Job PR-1184/9 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-1184/9/index.html

**Comment by mli** [comment]
Job PR-1184/10 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-1184/10/index.html

**Comment by mli** [comment]
Job PR-1184/11 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-1184/11/index.html

**Comment by mli** [comment]
Job PR-1184/12 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-1184/12/index.html

**Comment by mli** [comment]
Job PR-1184/13 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-1184/13/index.html

**Comment by StrayBird-ATSH** [comment]
Do you want to have reference for exactly every model(which will take more work and time), or I just add reference for those already mentioned in the corresponding page?

**Comment by mli** [comment]
Job PR-1184/14 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-1184/14/index.html

File: scripts/catalog.rst [hunk 5]
```diff
@@ -0,0 +1,336 @@
...
+--------------
+`Language Model Model Zoo Index <./language_model/index.html>`_
+
+Word Language Model
+~~~~~~~~~~~~~~~~~~~
+
+Dataset: Wikitext-2
+
++---------------------------------------+-----------------+-----------------------------------------------------------------------------------------------------------------------------+-----------------------------------------------------------------------------------------------------------------------------+
+| Pre-trained Model                     | Test Perplexity |Training Command                                                                                                             | log                                                                                                                         |
```
Comment (eric-haibin-lin) [inline scripts/catalog.rst:17]: Can we replace all `Training Command ` to `command`? For some tables, the first column is not wide enough.

Comment (StrayBird-ATSH) [reply scripts/catalog.rst:17]: OK. I will make corresponding changes by next commit

**Comment by StrayBird-ATSH** [comment]
Is it possible that we first merge this pull request and then I make updates in a new pull request? Since there has been a lot of updates on master branch, this might be less error-prone than merging a lot of commits to the current pull request.

**Comment by sxjscience** [comment]
I’m fine for merging this PR.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by szha** [comment]
@eric-haibin-lin ping for review

**Pull request merged**